### PR TITLE
refactor(node): use async await in fn main

### PR DESCRIPTION
**Pull request by cchen408**
# Description
- clean up lookup.js to use async await

### CODE REVIEW

File: traveler/examples/node/lookup.js [hunk 1]
```diff
@@ -116,18 +115,13 @@ function getLookupAddresses() {
...
-  tokenPromise.then(response => {
-    const accessToken = response.data.access_token;
-    const client = getClient(accessToken);
-
-    checkStatus(client);
-
-    const addresses = getLookupAddresses();
-    checkAddresses(client, addresses);
-  });
+async function main() {
```
Comment (bransorem) [inline traveler/examples/node/lookup.js:118]: Could use an IIFE, but this is more explicit about when `main` is being called.

Comment (cchen408) [reply traveler/examples/node/lookup.js:118]: went ahead and wrapped main

**Pull request merged**